components/director/internal/domain/package/service.go

```diff
@@ -161,3 +217,94 @@ func (s *service) ListByApplicationID(ctx context.Context, applicationID string,
 
 	return s.pkgRepo.ListByApplicationID(ctx, tnt, applicationID, pageSize, cursor)
 }
+
+func (s *service) createRelatedResources(ctx context.Context, in model.PackageCreateInput, tenant string, packageID string) error {
+	err := s.createAPIs(ctx, packageID, tenant, in.APIDefinitions)
```

Comment:
Hmm, but then we will bring more dependencies between services. I think that was what we wanted to avoid. I would keep it as it is for now (similarly what we have in Application service), but we definitely need to make a decision how to handle these dependencies.

At the beginning we handled deps like that: resolver -> service -> repo Now, we started to bring service to service deps. Let's think about it what's our next move.